pkg/cmd/render/render_test.go

```diff
@@ -214,6 +264,34 @@ func TestRenderIpv4(t *testing.T) {
 	testRender(config)
 }
 
+func TestRenderScalingStrategyBootstrapInPlace(t *testing.T) {
```

Comment:
(No action required) Maybe factor out the test configuration common to this new test and TestRenderIpv4?